.github/workflows/validate.yml

```diff
@@ -133,3 +133,20 @@ jobs:
       - name: Image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
 
+  coverage:
+    runs-on: ubuntu-latest
+    container: espressopp/buildenv:fedora
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Coverage build
+        run: |
+          cmake -B builddir -DUSE_GCOV=on -DCMAKE_BUILD_TYPE=Debug
+          cmake --build builddir --verbose -j 2
```

Comment:
general comment: Is the verbose output really beneficial? I think all the include paths etc hide the important information.

Reply:
I mainly wanted to make sure there is actually a `--coverage` flag.

I personally like to have verbose output so that all the information is there for debugging.